codex/manifest/types.nim

```diff
@@ -22,8 +24,22 @@ const
 type
   ManifestCoderType*[codec: static MultiCodec] = object
   DagPBCoder* = ManifestCoderType[multiCodec("dag-pb")]
+  VerificationHash* = F
```

Comment:
We cannot hardcode this, this needs to be parametrizable with the correct hash type.

Reply:
That makes perfect sense. I would like a nice generalized way we can wrap up the Poseidon2 field elements and plug them into the manifest, and drop the `VerificationHash` type completely.